Recrutify/Recrutify.Services/Services/FormEmailService.cs

```diff
@@ -1,36 +1,35 @@
 ﻿using System.Collections.Generic;
 using System.IO;
 using Mustache;
+using Recrutify.DataAccess;
+using Recrutify.DataAccess.Models;
 using Recrutify.Services.EmailModels;
 using Recrutify.Services.Services.Abstract;
 
 namespace Recrutify.Services.Services
 {
     public class FormEmailService : IFormEmailService
     {
-        public FormEmailService()
+        public IEnumerable<EmailRequest> GetEmailRequests(List<Candidate> candidates, Project project, string templatePath)
```

Comment:
think you can pass IEnumerable<Candidate> candidates, to be in one style with return type